src/asm/rgbasm.5

```diff
@@ -276,31 +294,28 @@ provide the interface to the section stack.
 will push the current section context on the section stack.
 .Ic POPS
 can then later be used to restore it.
-Useful for defining sections in included files when you don't want to destroy
-the section context for the program that included your file.
-The number of entries in the stack is limited only by the amount of memory in
-your machine.
+Useful for defining sections in included files when you don't want to destroy the section context for the program that included your file.
+The number of entries in the stack is limited only by the amount of memory in your machine.
 .Sh SYMBOLS
 .Pp
 .Ss Symbols
 RGBDS supports several types of symbols:
 .Pp
 .Bl -hang
 .It Sy Label
-Used to assign a memory location with a name
+Used to give a name to a memory location.
 .It Sy EQUate
 Give a constant a name.
-.It Sy SET
-Almost the same as EQUate, but you can change the value of a SET during
-assembling.
-.It Sy Structure Po Sy the RS group Pc
+.It Ic SET
+Almost the same as EQUate, but you can change the value of a SET during assembling.
+.It Sy Structure Pq Sy the RS group
 Define a structure easily.
-.It Sy String equate Pq Sy EQUS
+.It Sy String equate Pq Ic EQUS
 Give a frequently used string a name.
 Can also be used as a mini-macro, like
-.Fd #define
+.Cm #define
```

Comment:
`Fd` actually exists specifically for C preprocessor directives, so this should be reverted.

Reply:
Okay